node/subsystem/src/messages.rs

```diff
@@ -536,7 +536,7 @@ impl ProvisionerMessage {
 		match self {
 			Self::RequestBlockAuthorshipData(hash, _) => Some(*hash),
 			Self::RequestInherentData(hash, _) => Some(*hash),
-			Self::ProvisionableData(_) => None,
+			Self::ProvisionableData(hash, _) => Some(*hash),
```

Comment:
Given that most variants of `ProvisionableData` include a `Hash`, might it make more sense to just delegate to that hash instead of adding a new one to this enum?

https://github.com/paritytech/polkadot/blob/23d3de11a63b5365b7dc88920086bb5211e7306f/node/subsystem/src/messages.rs#L499-L508

Reply:
Yeah I thought the same, but I wanted to make it more explicit here. I can also change it